### PR TITLE
Rename lsra temps with meaningful names.

### PR DESCRIPTION
**Pull request by sandreenko**
It was hard to follow the logic.

### CODE REVIEW

**Comment by sandreenko** [comment]
PTAL @CarolEidt. @dotnet/jit-contrib.
I am not sure that I chose the best names, so if you have a better idea, please write them and I will apply it.

File: src/jit/lsra.cpp [hunk 1]
```diff
@@ -2607,44 +2607,39 @@ void LinearScan::checkLastUses(BasicBlock* block)
...
         }
 
         assert(currentRefPosition != refPositions.rend());
         ++currentRefPosition;
     }
 
-    VARSET_TP temp2(VarSetOps::MakeCopy(compiler, block->bbLiveIn));
-    VarSetOps::DiffD(compiler, temp2, temp);
-    VarSetOps::DiffD(compiler, temp, block->bbLiveIn);
+    VARSET_TP usedAsDead(VarSetOps::Diff(compiler, block->bbLiveIn, lastUses));
```
Comment (CarolEidt) [inline src/jit/lsra.cpp]: These names are more confusing than enlightening to me. Perhaps `usedAsDead` could be better named as `liveInNotComputedLive` and `liveNotUsed` as `complutedLiveNotLiveIn`.

Comment (JosephTremoulet) [inline src/jit/lsra.cpp]: and `lastUses` -> `computedLive` accordingly

Comment (sandreenko) [inline src/jit/lsra.cpp]: Fixed, please take another look.

File: src/jit/lsra.cpp [hunk 2]
```diff
@@ -2545,11 +2545,11 @@ void LinearScan::checkLastUses(BasicBlock* block)
     // find which uses are lastUses
 
     // Work backwards starting with live out.
-    // 'temp' is updated to include any exposed use (including those in this
+    // 'lastUses' is updated to include any exposed use (including those in this
```
Comment (CarolEidt) [inline src/jit/lsra.cpp]: This should now be 'computedLive'.

**Comment by sandreenko** [comment]
@dotnet-bot test Tizen armel Cross Debug Build

**Comment by sandreenko** [comment]
@dotnet-bot test Tizen armel Cross Release Build

**Comment by sandreenko** [comment]
@dotnet-bot test Windows_NT x64 Debug Build and Test

**Comment by sandreenko** [comment]
@dotnet-bot test OSX10.12 x64 Checked Build and Test 
@dotnet-bot test Ubuntu x64 Checked Build and Test
@dotnet-bot test Windows_NT x64 Debug Build and Test

**Comment by sandreenko** [comment]
@dotnet-bot test Windows_NT x64 Debug Build and Test
It looks like an infrastructure problem. I am lucky today.
